tm2py/data_models/maz_data.py

```diff
@@ -0,0 +1,325 @@
+        util (int): Utilities employment
+        emp_total (int); Total employment
+        publicEnrollGradeKto8 (int): Public enrollment grades k-8
+        privateEnrollGradeKto8 (int): Private enrollment grades k-8
+        publicEnrollGrade9to12 (int): Public enrollment grades 9-12
+        privateEnrollGrade9to12 (int): Private enrollment grades 9-12
+        comm_coll_enroll (int): Community college enrollment
+        EnrollGradeKto8 (int): Total enrollment grades k-8
+        EnrollGrade9to12 (float): Total enrollment grades 9-12
+        collegeEnroll (float): Major College enrollment
+        otherCollegeEnroll (float): Other College enrollment
+        AdultSchEnrl (int): Adult School enrollment 
+        hstallsoth (float): Number of stalls allowing hourly parking for trips with destinations in other MAZs
+        hstallssam (float): Number of stalls allowing hourly parking for trips with destinations in the same MAZ
+        dstallsoth (float): Stalls allowing daily parking for trips with destinations in other MAZs
+        dstallssam (float): Stalls allowing daily parking for trips with destinations in the same MAZ
+        mstallsoth (float): Stalls allowing monthly parking for trips with destinations in other MAZs
+        mstallssam (float): Stalls allowing monthly parking for trips with destinations in the same MAZ
+        park_area (float): Area of parks in sq. meters
+        hparkcost (float): Average cost of parking for one hour in hourly stalls in this MAZ, dollars
+        numfreehrs (float): Number of hours of free parking allowed before parking charges begin in hourly stalls
+        dparkcost (float): Average cost of parking for one day in daily stalls, dollars
+        mparkcost (float): Average cost of parking for one day in monthly stalls, amortized over 22 workdays, dollars
+        ech_dist (int): Elementary school district
+        hch_dist (int): High school district
+        parkarea (int): parking area (1 through 4)
+        TERMINAL (float):
+        MAZ_X (float):
+        MAZ_Y (float):
+        TotInt (float): Total intersections within 1/2 mile of MAZ
+        EmpDen (float): Employment per acre within 1/2 mile of MAZ
+        RetEmpDen (float): Retail employment per acre within 1/2 mile of MAZ
+        DUDen (float): Households per acre within 1/2 mile of MAZ
+        PopDen (float): Population per acre within 1/2 mile of MAZ
+        IntDenBin (int): Intersection density bin (1 through 3 where 3 is the highest)
+        EmpDenBin (int): Employment density bin (1 through 3 where 3 is the highest)
+        DUDenBin (int): Houseold density bin (1 through 3 where 3 is the highest)
+        PopEmpDenPerMi (float):
+    """
+    MAZ: Series[int] = Field(nullable=False, unique=True)
+    TAZ: Series[int] = Field(nullable=False)
+    MAZ_ORIGINAL: Series[int] = Field(nullable=False, unique=True)    
+    TAZ_ORIGINAL: Series[int] = Field(nullable=False)
+    DistID: Series[int] = Field(nullable=False)
```

Comment:
These should only include `MAZ_NODE` and `TAZ_NODE` (and maybe `MAZ_SEQ` and `TAZ_SEQ` if it's in the maz data...)
We don't want to reference `MAZ` or `TAZ` without the suffix because it's ambiguous.
And the `_ORIGINAL` terms are deprecated.